### PR TITLE
Moving the Clojure Camp website from Javascript to Clojure with Babashka

### PR DESCRIPTION
**Pull request by taulant**
- Using Hiccup to draft and generating html pages
- Generating an index.html file with spit command
- Using `bb start` to build and locally serve the website

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1,3 @@
...
-.env.development.local
-.env.test.local
-.env.production.local
-
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+.clj-kondo
+.lsp
+.calva
```
Comment (rafd) [inline .gitignore]: .lsp and .calva are ignores specific to your dev tooling not the project. I would put these in your global .gitignore.

File: src/run.clj [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(ns run)
```
Comment (rafd) [inline src/run.clj]: I would put these under a namespace like `(ns clojure-camp-site.run)`

File: src/run.clj [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+(require '[root]
+         '[babashka.http-server :as http])
+
+(defn build [content] (spit "static/index.html" content))
+
+(defn serve [] ((build root/html)
```
Comment (rafd) [inline src/run.clj]: Is there a `do` missing here?

File: static/index.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html class="foo"><head><meta><title>Clojure Camp</title></meta><script src="https://cdn.tailwindcss.com"></script></head><body class="m-8"><h1 class="my-8 text-4xl">Clojure Camp</h1><a class="my-4 p-4 border-2 block-inline" href="http://handbook.clojure.camp" target="_blank">Read the Handbook</a></body></html>
```
Comment (rafd) [inline static/index.html]: Ideally, we'll have a github build script generate this rather than committing.

File: src/run.clj (hunk 2 above)
Comment (taulant) [reply src/run.clj]: @rafd Updated. I also had to update folder structure for this but I think it should be fine

**Pull request merged**